packages/base-map/src/index.story.tsx

```diff
@@ -1,6 +1,12 @@
 /* eslint-disable react/button-has-type */
 import React, { useRef } from "react";
-import { MapboxMap, Popup } from "react-map-gl";
+import {
+  MapboxMap,
```

Comment:
Sort imports

Reply:
Sorted in 82cf6e389b83c136c2d8370515fdbcfead3294b6